services/frontend-v3/src/i18n/fr_CA.json

```
@@ -389,6 +397,8 @@
   "profile.visibleCards.friends": "Amis seulement",
   "profile.visibleCards.private": "Privée",
 
+  "tooltip.extra.info.help": "Pour plus d'informations, visitez notre page d'aide et de contact: ",
```

In French you would add a space before the :
```suggestion
  "tooltip.extra.info.help": "Pour plus d'informations, visitez notre page d'aide et de contact : ",
```